pkg/util/healthcheck/healthchecker.go

```diff
@@ -35,14 +36,14 @@ func init() {
 		WithComponent("healthChecker")
 }
 
-//StartPeriodicHealthCheck - starts a job that runs the periodic healthchecks
+// StartPeriodicHealthCheck - starts a job that runs the periodic health checks
 func StartPeriodicHealthCheck() {
 	interval := defaultCheckInterval
 	if GetStatusConfig() != nil {
 		interval = GetStatusConfig().GetHealthCheckInterval()
 	}
 	periodicHealthCheckJob := &periodicHealthCheck{interval: interval}
-	go periodicHealthCheckJob.Execute()
+	jobs.RegisterIntervalJobWithName(periodicHealthCheckJob, periodicHealthCheckJob.interval, "Periodic Health Check")
```

Comment:
Since all this does is see if the health check is still good, can't it run every second, for instance? To quickly bring down the jobs when a failure occurs?

Reply:
Is there any reason to have the health check interval config value then?

Reply:
My bad, I was thinking that was your new job.  This is fine.